README.md

```diff
@@ -1,3 +1,109 @@
-# final-project
+# StudyBuddy
 
-A full stack JavaScript solo project.
+A full-stack web application for students to create and study flashcards.
+
+## :thinking: Why I built this
+
+While attending LearningFuze's Accelerated Full Immersion Web Development Bootcamp I found myself using
+apps like Brainscapes and Quizlet to study the information I was learning. I wanted to create a flash card
+app for myself and future students to use and study with. I also wanted to create something that felt more
+immersive with a classroom/supplies theme and CSS animations. The buddy and instructor cats in the app are based on my
+two cats, Bacchus and Apollo!
+
+### 🔗 Live Demo
+
+Try the application live here: https://study-buddy.brandonmoy.com/
+
+## 💻 Technologies Used
+
+### Languages
+- HTML5
+- CSS3
+- JavaScript (ES6)
+- React
+- JSX
+- Node.js
+- Express.js
+- Babel
+- Webpack
+- Argon2
+- JSON Web Token
+- Dotenv
```

Comment:
I recommend distinguishing between Languages and Packages if you want to include both